Header header

bool     reload_all
string   topic   # topic to reload, if reload_all is false